api_tests/e2e/rfc003/btc_eth/inverted_refund.ts

```diff
@@ -1,122 +1,46 @@
-import { expect } from "chai";
-import "chai/register-should";
-import { ethers } from "ethers";
-import { Actor } from "../../../lib/actor";
-import * as bitcoin from "../../../lib/bitcoin";
-import { ActionKind, SwapRequest } from "../../../lib/comit";
-import "../../../lib/setup_chai";
-import { createTests, Step } from "../../../lib/test_creator";
-import { HarnessGlobal } from "../../../lib/util";
-
-declare var global: HarnessGlobal;
-
-(async function() {
-    const alice = new Actor("alice", {
-        ledgerConfig: global.ledgerConfigs,
-        addressForIncomingBitcoinPayments:
-            "bcrt1qs2aderg3whgu0m8uadn6dwxjf7j3wx97kk2qqtrum89pmfcxknhsf89pj0",
-    });
-    const bob = new Actor("bob", {
-        ledgerConfig: global.ledgerConfigs,
-        addressForIncomingBitcoinPayments: null,
+import { Actor } from "../../../lib_sdk/actors/actor";
```

Comment:
I know it's not changed here but the name of the file `inverted_refund` is very cryptic.

Reply:
Do you have an alternative proposal?

Reply:
no 👼